test/integration/test.txt

```diff
@@ -0,0 +1 @@
+Test
```

Comment:
Please see my comment above on creating a new file in the bucket instead of uploading one. If we must upload a file, let's find a better name for this. `storage_test_file.txt` or something... and it should be in the `test/resources` folder.